news/bugfix-4689.md

```diff
@@ -0,0 +1,4 @@
+    cfg-parser: fix buffer overflow in case flag was long
```

Comment:
I think this entry is not useful for users. Instead we should mention that we fixed a crash which occurred for long strings passed to the scope() option.

Reply:
Yeah, reworded it, thanks!